Binary files xalan-j_2_2_0.orig/.build.xml.swp and xalan-j_2_2_0/.build.xml.swp differ
diff -Nur xalan-j_2_2_0.orig/build.sh xalan-j_2_2_0/build.sh
--- xalan-j_2_2_0.orig/build.sh	Tue Jan 15 22:30:02 2002
+++ xalan-j_2_2_0/build.sh	Mon Jan 28 16:33:58 2002
@@ -21,7 +21,7 @@
     ANT_JAR=./bin/ant.jar
 fi
 if [ "$PARSER_JAR" = "" ] ; then
-    PARSER_JAR=./bin/xerces.jar
+    PARSER_JAR=/usr/lib/java/xerces.jar
 fi
 
 if [ "$XML-APIS_JAR" = "" ] ; then
